rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-sts-creating-a-cluster-ext-auth"]
+= Creating ROSA with HCP clusters with external authentication
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-sts-creating-a-cluster-ext-auth
+
+toc::[]
+
+You can create {hcp-title-first} clusters that use external authentication to issue your access tokens.
+
+[IMPORTANT]
+====
+Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must create a new cluster to use {hcp-title} functionality. You also cannot convert a cluster that was created to use external authentication providers to use the internal OAuth2 server. You must also create a new cluster.
+====
+
+[NOTE]
+====
+{hcp-title} clusters only support {sts-first} authentication.
+====
+
+.Further reading
+* For a comparison between {hcp-title} and ROSA Classic, see the xref:../architecture/rosa-architecture-models.adoc#rosa-hcp-classic-comparison_rosa-architecture-models[Comparing architecture models] documentation.
+* See the AWS documentation for information about link:https://docs.aws.amazon.com/rosa/latest/userguide/getting-started-hcp.html[Getting started with ROSA with HCP using the ROSA CLI in auto mode].
+
+.Additional resources
+
+For a full list of the supported certificates, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-policy-process-security.adoc#rosa-policy-compliance_rosa-policy-process-security[Compliance] section of "Understanding process and security for Red Hat OpenShift Service on AWS".
+
+[id="rosa-hcp-external-auth-prereqs"]
+== {hcp-title} Prerequisites
+
+To create a {hcp-title} cluster, you must have:
```

Comment:
The [lead-in to a list](https://www.ibm.com/docs/en/ibm-style?topic=format-lists) should be a complete sentence it its own right rather than using the list items to complete it